### PR TITLE
Add landing page convention

### PR DESCRIPTION
**Pull request by saivickna**
Why?

Useful for showing links to relevant configs and provide information about a service. Works in tandem with health check convention and config convention.
Can specify a description or github link by by binding 'description' or 'github' to the microcosm services config

### CODE REVIEW

File: microcosm_flask/conventions/landing.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    @graph.flask.route("/")
+    def render_landing_page():
+        """
+        Render landing page
+
+        """
+        return Template(template).render(
+            service_name=graph.metadata.name,
+            swagger_version=graph.config.swagger_convention.version,
+            description=graph.config.get('description', None),
```
Comment (jessemyers) [inline microcosm_flask/conventions/landing.py]: This is a good start, but I'd love to make this not require configuration.

One possible direction:
 - See if we can programmatically inspect any of the data set in `setup.py` (e.g. `long_description`)
 - Write something to populate that field in `setup.py` from a `README.md` (and move developer information into a different file, e.g `CONTRIBUTING.md` or `DEVELOPING.md`)

File: microcosm_flask/conventions/landing.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    @graph.flask.route("/")
+    def render_landing_page():
+        """
+        Render landing page
+
+        """
+        return Template(template).render(
+            service_name=graph.metadata.name,
+            swagger_version=graph.config.swagger_convention.version,
```
Comment (jessemyers) [inline microcosm_flask/conventions/landing.py]: What if there are multiple versions at once?

We've been discouraging the use of `graph.config.swagger_convention.version` in many/most of our services for this reason.

File: microcosm_flask/conventions/landing.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            description=graph.config.get('description', None),
+            github=graph.config.get('github', None),
```
Comment (jessemyers) [inline microcosm_flask/conventions/landing.py]: As above, probably available in `setup.py`, might be accessible programmatically.

File: microcosm_flask/conventions/landing.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        """
+        properties = get_properties()
+        description = get_description(properties)
+        swagger_versions = get_swagger_versions()
+
+        return Template(template).render(
+            service_name=graph.metadata.name,
+            swagger_versions=swagger_versions,
+            description=description,
+            github=properties.get("Home-page"),
```
Comment (pelegdvir) [inline microcosm_flask/conventions/landing.py]: Use Home instead of github?

**Pull request merged**